packages/common/src/hooks/useCoinflowAdapter.ts

```diff
@@ -122,15 +127,36 @@ export const useCoinflowAdapter = () => {
         connection,
         wallet: {
           publicKey: wallet.publicKey,
-          sendTransaction: async (transaction: Transaction) => {
-            transaction.partialSign(wallet)
-            if (isUseSDKPurchaseTrackEnabled) {
+          sendTransaction: async (
+            transaction: Transaction | VersionedTransaction
+          ) => {
+            if (
+              isUseSDKPurchaseTrackEnabled &&
+              transaction instanceof VersionedTransaction
+            ) {
               const sdk = await audiusSdk()
+
+              // Get a more recent blockhash to prevent BlockhashNotFound errors
```

Comment:
comment is a little weird because maybe it's a less recent blockhash?

Reply:
It's almost certainly more recent - we're making the same `getLatestBlockhash` call with the same RPC at a later time